### PR TITLE
add annotations to fix scorecard tests

### PR DESCRIPTION
**Pull request by nishant-parekh**
Change-Id: I790874b2c883de1f377809af80ea815b98e66587
Signed-off-by: Nishant Parekh <nparekh@redhat.com>

### CODE REVIEW

File: bundle/manifests/event.redhat-cne.org_hardwareevents.yaml [hunk 1]
```diff
@@ -59,7 +59,7 @@ spec:
               lastSyncTimestamp:
                 description: 'INSERT ADDITIONAL STATUS FIELD - define observed state
                   of cluster Important: Run "make" to regenerate code after modifying
-                  this file'
+                  this file LastSynced time of the custom resource'
```
Comment (aneeshkp) [inline bundle/manifests/event.redhat-cne.org_hardwareevents.yaml:62]: Description wrong

Comment (nishant-parekh) [reply bundle/manifests/event.redhat-cne.org_hardwareevents.yaml]: fixed

**Review by jzding — APPROVED** [review]

**Pull request merged**